### PR TITLE
Clean up DateUtils API

### PR DESCRIPTION
**Pull request by cristianonicolai**

### CODE REVIEW

File: jbpm-wb-common/jbpm-wb-common-client/src/test/java/org/jbpm/workbench/common/client/util/DateUtilsTest.java [hunk 1]
```diff
@@ -64,4 +65,22 @@ public void testGetDateAndTimeStr() {
         assertEquals("",
                      DateUtils.getDateTimeStr(null));
     }
+
+    @Test
+    public void testCreateDateWithDefaultFormat() {
+        Date date = createDate("2013-05-01");
+        System.out.println(date);
```
Comment (bsremac) [inline jbpm-wb-common/jbpm-wb-common-client/src/test/java/org/jbpm/workbench/common/client/util/DateUtilsTest.java]: I think you unintentionally left this print statement in.

File: jbpm-wb-common/jbpm-wb-common-client/src/main/java/org/jbpm/workbench/common/client/util/DateUtils.java [hunk 2]
```diff
@@ -19,7 +19,6 @@
 import java.util.Date;
```
Comment (bsremac) [inline jbpm-wb-common/jbpm-wb-common-client/src/main/java/org/jbpm/workbench/common/client/util/DateUtils.java]: I guess we should also consider moving to ```java.time``` package in the future, as ```java.util.Date``` looks deprecated

File: jbpm-wb-common/jbpm-wb-common-client/src/test/java/org/jbpm/workbench/common/client/util/DateUtilsTest.java (hunk 1 above)
Comment (cristianonicolai) [inline jbpm-wb-common/jbpm-wb-common-client/src/test/java/org/jbpm/workbench/common/client/util/DateUtilsTest.java]: Copied it over from the deleted test class, fixed now.

File: jbpm-wb-common/jbpm-wb-common-client/src/main/java/org/jbpm/workbench/common/client/util/DateUtils.java (hunk 2 above)
Comment (cristianonicolai) [inline jbpm-wb-common/jbpm-wb-common-client/src/main/java/org/jbpm/workbench/common/client/util/DateUtils.java]: Definitely.

Comment (manstis) [inline jbpm-wb-common/jbpm-wb-common-client/src/main/java/org/jbpm/workbench/common/client/util/DateUtils.java]: IIRC ```java.time``` is unsupported by GWT at present.

**Comment by bsremac** [comment]
+1

**Pull request merged**